torch/testing/_internal/opinfo/definitions/nested.py

```diff
@@ -418,6 +422,41 @@ def sample_inputs_masked_select(
         )
 
 
+def sample_inputs_index_put(
+    op_info, device, dtype, requires_grad, op_kwargs=None, **kwargs
+):
+    for njt in _sample_njts(
+        device=device, dtype=dtype, requires_grad=requires_grad, dims=[2]
```

Comment:
I think it's worth testing dims 3 and 4 too, especially for the case where indices are specified only for the earlier dims